Source/WarpX.cpp

```diff
@@ -976,12 +975,6 @@ WarpX::ReadParameters ()
                 "psatd.update_with_rho must be equal to 1 for comoving PSATD");
         }
 
-        constexpr int zdir = AMREX_SPACEDIM - 1;
```

Comment:
This is the place where I would put the checking the damped boundary conditions, making sure that it is consistent and correct.

Reply:
Done!